src/components/DocumentRenderer/DocumentDetailsSheet.test.tsx

```diff
@@ -1,14 +1,35 @@
 import React from "react";
-import { render } from "@testing-library/react-native";
+import { render, wait } from "@testing-library/react-native";
 import sampleDoc from "../../../fixtures/demo-oc.json";
 import { DocumentDetailsSheet } from "./DocumentDetailsSheet";
 
+import { checkValidity } from "../../services/DocumentVerifier";
+jest.mock("../../services/DocumentVerifier");
+const checkValidityMock = checkValidity as jest.Mock;
+
+jest.useFakeTimers();
+
 describe("DocumentDetailsSheet", () => {
-  it("should show the correct issuer name", () => {
+  it("should show the correct issuer name", async () => {
     expect.assertions(1);
+    checkValidityMock.mockReturnValue([
+      Promise.resolve({
+        hash: { checksumMatch: true },
+        issued: { issuedOnAll: true },
+        revoked: { revokedOnAny: false },
+        valid: true
+      }),
+
+      Promise.resolve({
+        identifiedOnAll: true,
+        details: []
```

Comment:
What is expected to be in this details?

Reply:
It's the details of each of the issuer, taken from https://github.com/TradeTrust/tradetrust-website/blob/master/src/services/verify/index.js